src/assets/toolkit/scripts/toolkit.js

```diff
@@ -9,9 +9,14 @@ import {arrayFromSelector} from './lib/dom/core';
 import {FloatLabel} from './lib/component/float-label';
 import {Sky} from './lib/component/sky';
 import {ElasticTextarea} from './lib/component/elastic-textarea';
+import {FactSlider} from './lib/component/fact-slider';
 
 (function() {
 
+  arrayFromSelector('.FunFacts').map(element => {
```

Comment:
See [this comment](https://github.com/cloudfour/cloudfour.com-patterns/pull/223/files#r66166215).